### ISSUE TITLE
Error getting repository data for C7.6.1810-base

### ISSUE THREAD

**Issue by timohirt** [dcos/dcos-e2e#1585]
Can't create a mesos cluster usinfg vagrant on OSX 10.14.2. Tried VirtualBox 6.0 and 5.2.24. Seems that OSX doesn't support 5.1.x anymore.

```bash
$ vagrant --version
Vagrant 2.2.3
```

Installation fails with the following error message:

```bash
================================================================================
 Package              Arch         Version                     Repository  Size
================================================================================
Updating:
 centos-release       x86_64       7-6.1810.2.el7.centos       base        26 k

Transaction Summary
================================================================================
Upgrade  1 Package

Total download size: 26 k
Downloading packages:
Delta RPMs disabled because /usr/bin/applydeltarpm not installed.
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Updating   : centos-release-7-6.1810.2.el7.centos.x86_64                  1/2
  Cleanup    : centos-release-7-3.1611.el7.centos.x86_64                    2/2
  Verifying  : centos-release-7-6.1810.2.el7.centos.x86_64                  1/2
  Verifying  : centos-release-7-3.1611.el7.centos.x86_64                    2/2

Updated:
  centos-release.x86_64 0:7-6.1810.2.el7.centos

Complete!
Loaded plugins: fastestmirror


Error getting repository data for C7.6.1810-base, repository not found
Error creating cluster.
```

How to fix this? Someone also asked on Stackoverflow: https://stackoverflow.com/questions/54121527/vagrant-error-getting-repository-data-for-c7-6-1810-base-repository-not-found

**Comment by rajcspsg** [comment]
@timohirt I don't know whether you are the one who answered in comment to the above question in SO. The answer is now updated there.

**Comment by adamtheturtle** [comment]
There is now a new version of vagrant-vbguest which fixes the issue (0.17.2).
There is also a new version of minidcos which includes a doctor check that at least this version is installed.

**Comment by timohirt** [comment]
Thanks @adamtheturtle !
